lib/_debug_agent.js

```diff
@@ -185,6 +195,11 @@ Client.prototype.onCommand = function onCommand(cmd) {
   this.binding.sendCommand(cmd.body);
 
   this.agent.notifyWait();
+  
+  if (cmd.body == disconnectRequest) {
```

Comment:
I won't use JSON.parse here because I'm worried about performance.

So, how I can understand you worried about situation, when we receive from user a `disconnect` request with other order of serialized properties. 
But the most bad that can happens in this situation - we'll send two `disconnect` requests to debugger (one real from user and one other generated by `debug-agent` script). This behavior also exists in old debugger.

Unless you want to send two `disconnect` requests to debugger, I will think how to avoid this situation with minimal performance loss.